spaces/S000110/properties/P000136.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000110
+property: P000136
+value: true
+---
+
+Let $K \subset X$ be compact. If $K$ contains infinitely many elements of $M$, then $\{\omega \cup \{F\}: F \in M \cap K\}$ is an open cover with no finite subcover. Thus, $K$ contains finitely many elements of $M$, which we can list as $F_1, \cdots, F_n$.
+
+Now, suppose $K$ contains infinitely many elements of $\omega$. We inductively choose $A_i \in F_i$ for $1 \leq i \leq n$ as follows: We shall choose our indutive assumption as $B_i = (\omega \cap K) \setminus (A_1 \cup \cdots \cup A_{i-1})$ being infinite. Write $B_i = C_i \cup D_i$ for two infinite, disjoint $C_i$ and $D_i$. If $C_i \in F_i$, let $A_i = C_i$. Otherwise, let $A_i = \omega \setminus C_i$. We note that, either way, $B_{i+1} = B_i \setminus A_i$ is infinite.
```

Comment:
```suggestion
Now, suppose $K$ contains infinitely many elements of $\omega$. We inductively choose $A_i \in F_i$ for $1 \leq i \leq n$ as follows: We let $B_1=\omega\cap K$ and shall choose our inductive assumption as $B_i = (\omega \cap K) \setminus (A_1 \cup \cdots \cup A_{i-1})$ being infinite. Write $B_i = C_i \cup D_i$ for two infinite, disjoint $C_i$ and $D_i$. If $C_i \in F_i$, let $A_i = C_i$. Otherwise, let $A_i = \omega \setminus C_i$, which necessarily belongs to $F_i$. We note that, either way, $B_{i+1} = B_i \setminus A_i$ is infinite.
```
minor clarification